Assets/Scripts/Game/Serialization/SaveLoadManager.cs

```diff
@@ -754,10 +803,31 @@ SaveData_v1 BuildSaveData()
             saveData.sceneCache = stateManager.GetSceneCache();
             saveData.travelMapData = DaggerfallUI.Instance.DfTravelMapWindow.GetTravelMapSaveData();
             saveData.advancedClimbingState = GameManager.Instance.ClimbingMotor.GetSaveData();
+            saveData.modInfoData = GetModInfoData();
 
             return saveData;
         }
 
+        ModInfo_v1[] GetModInfoData()
+        {
+            List<ModInfo_v1> records = new List<ModInfo_v1>();
+            foreach (var mod in ModManager.Instance.GetAllMods())
```

Comment:
You can use the property _Mods_ instead of _GetAllMods()_ to avoid creating a new collection.

Reply:
Resolved!